low-level/framework/utils/utility.py

```diff
@@ -156,6 +158,56 @@ def get_current_time():
         """Returns the time as integer number in seconds since the epoch in UTC."""
         return int(time.time())
 
+    @staticmethod
```

Comment:
Create new file and class, & put these function there, like in low-level/framework/platforms/server/platform.py
class Platform

Reply:
This will be addressed.

Reply:
Done.